packages/types/src/tx/schema/transfer.ts

```diff
@@ -17,7 +17,16 @@ export class TransparentTransferMsgValue {
   @field(BigNumberSerializer)
   amount!: BigNumber;
 
-  constructor(data: TransparentTransferProps) {
+  constructor(data: TransferDataProps) {
+    Object.assign(this, data);
+  }
+}
+
+export class TransferMsgValue {
+  @field({ type: vec(TransferDataMsgValue) })
+  data!: TransferDataMsgValue[];
+
+  constructor(data: TransferProps) {
     Object.assign(this, data);
```

Comment:
The serialization of this is not working for me when I call `buildTransparentTransfer`. It looks like the problem is the data needs to be mapped with `new TransferDataMsgValue` to add the borsh methods to the plain data objects.

Reply:
Interesting, I'll take a look! I did not test the whole process, just tried to get types updated so it will build, but should be easy enough to fix!

Reply:
Fixed!